content/en/_redirects

```diff
@@ -70,7 +70,6 @@
 docs/started/requirements/                    /docs/started/getting-started/
 
 # Created a new components layout
-/docs/components/hyperparameter                /docs/components/katib/overview
```

Comment:
It's usually best to leave the old redirects, as it creates history, and is fine to chain them.

Reply:
@thesuperzapper I didn't remove the old redirect, I just moved it below: https://github.com/kubeflow/website/blob/817621ab42cae173a4ee92aba39b417967ee73f4/content/en/_redirects#L238

Reply:
@andreyvelich yea, don't do that, as it might break a subsequent redirects that have happened since (if the page has been moved multiple times).

We should only add things to the end of the file. That lets us assume that the current state of the website already has its redirects set up, so we only add new redirects for the actual changes we have made.